stripe/api_resources/transfer.py

```diff
@@ -18,34 +17,3 @@ class Transfer(
     UpdateableAPIResource,
 ):
     OBJECT_NAME = "transfer"
-
-    @classmethod
-    def _cls_cancel(
```

Comment:
Why was this removed in that commit? Should this be in the changelog?

Reply:
I can open a separate PR with those changes. This is the result of me running the latest generator against the latest  openapi from `sdk-release/next-major`